src/aws/k8s-empty/k8s_module/main.tf

```diff
@@ -110,7 +110,7 @@ resource "aws_instance" "control-plane" {
   }
 
   provisioner "file" {
-    source      = "../../../../k8s-labs/src/kubeadm/src/aws/manifests"
+    source      = "../../../../k8s-labs/src/kubeadm/src/manifests/aws"
```

Comment:
Hmm, there is no `aws` folder under `k8s-labs/src/kubeadm/src/manifests`. Maybe a typo?